feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -49,61 +49,77 @@
       </div>
     </div>
 
-    <div class="cx-product-card-rows">
-      <cx-configurator-attribute-quantity
-        (changeQuantity)="onChangeQuantity($event)"
-        [allowZero]="!preventAction"
-        [initialQuantity]="product.quantity"
-        [disableQuantityActions]="loading$ | async"
-        *ngIf="showQuantity"
-        class="cx-product-card-quantity"
-      ></cx-configurator-attribute-quantity>
-
-      <div class="cx-product-card-price"></div>
-    </div>
-
-    <div class="cx-product-card-rows" *ngIf="!singleDropdown">
+    <div
+      class="cx-product-card-rows column"
+      *ngIf="!singleDropdown || getProductPrice(product)"
```

Comment:
the component is aware of the product, so we do not need to pass the product as parameter?

Reply:
Done